lib/importers/category_importer.rb

```diff
@@ -22,6 +23,16 @@ def page_titles_for_category(category, depth=0, namespace=nil)
     page_properties_for_category(category, 'title', depth, namespace)
   end
 
+  def page_titles_for_psid(psid)
```

Comment:
I don't think this makes sense as part of CategoryImporter, as it's doing something fundamentally different from CategoryImporter, which is all about getting data from a specific named category via MediaWiki.

Although PetScan does use categories itself, from the Dashboard's perspective it's a whole different source of data.

It's simple enough, though, that you might just add this method into `PetScanApi` directly.